### ISSUE TITLE
Add utils to export weights to jax format

### ISSUE THREAD

**Issue by Luofan-KK** [aqlaboratory/openfold#333]
OpenFold is a great platform to train model. I've trained a satisfied model. Considering that AlphaFold and ColabFold are still the mostly used platform, I want to export my pytorch weights to JAX format weights, so users don't need to prepare a new env to use my model.
Is it possible? I've read "import_weights.py", but lost at "assign" function and ParamType function.
Thanks

**Comment by Luofan-KK** [comment]
I just get an Idea. Although I can't read "assign" and "ParamType" function, I can just convert jax param to torch tensor format, then modify "copy_" to "-", then use the difference to calculate gradient and train jax param.
